### PR TITLE
Snapshot DB migration

### PR DESCRIPTION
**Pull request by rhbntt**

### CODE REVIEW

File: core/node/storage/pg_stream_store_test.go [hunk 1]
```diff
@@ -1241,12 +1233,13 @@ func TestReadStreamFromLastSnapshot(t *testing.T) {
...
-		Hash:   mb.Hash,
-		Data:   mb.Data,
+		Number:   mb.Number,
+		Hash:     mb.Hash,
+		Data:     mb.Data,
+		Snapshot: []byte("snapshot2"),
 	}))
 	lastEvents = dataMaker.events(5)
-	require.NoError(promoteMiniblockCandidate(ctx, pgStreamStore, streamId, mb.Number, mb.Hash, true, lastEvents))
+	require.NoError(promoteMiniblockCandidate(ctx, pgStreamStore, streamId, mb.Number, mb.Hash, lastEvents))
```
Comment (sergekh2) [inline core/node/storage/pg_stream_store_test.go:1242]: it's hard to say, are there now tests that also test nil snapshots? if not, please add.

**Review by sergekh2 — APPROVED** [review]
